### PR TITLE
Ingested MT Summit 2003.

### PR DESCRIPTION
**Pull request by davidstap**
See https://aclanthology.org/www.mt-archive.info/00/MTS-2003-TOC.htm for original.

### CODE REVIEW

File: data/xml/2003.mtsummit.xml [hunk 1]
```diff
@@ -0,0 +1,857 @@
...
+  <volume id="plenaries" ingest-date="2021-07-23">
+    <meta>
+      <booktitle>Proceedings of Machine Translation Summit IX: Plenaries</booktitle>
+      <address>New Orleans, USA</address>
+      <month>September 23-27</month>
+      <year>2003</year>
+    </meta>
+    <frontmatter>
+      <url hash="e349b78d">2003.mtsummit-plenaries.0</url>
+    </frontmatter>
```
Comment (mjpost) [inline data/xml/2003.mtsummit.xml:12]: What is in this PDF? You could upload the PDFs so the links would work.

Related, I think: where are the "Welcome" and "Preface" from Hovy and Macklovitch?

**Review by mjpost — APPROVED** [review]
This looks good from a cursory pass. One comment, is that for keynotes, I think the slides should be added as a Presentation attachment, instead of as a PDF, unless the talk is given on a particular paper. Can you make this switch?

File: data/xml/2003.mtsummit.xml (hunk 1 above)
Comment (davidstap) [reply data/xml/2003.mtsummit.xml:12]: Strange, I have already uploaded the PDFs (to aclanthology.org). I have combined the "Welcome" and "Preface" into a single PDF and added this as frontmatter, what do you think?

Comment (mjpost) [reply data/xml/2003.mtsummit.xml:12]: They must have got deleted with the syncing bug. Can you re-upload?

Comment (davidstap) [reply data/xml/2003.mtsummit.xml:13]: Everything is re-uploaded now.

**Pull request merged**